docs/AWSchatbot.md

```diff
@@ -0,0 +1,18 @@
+# Setting up notifications from AWS to Slack
```

Comment:
This looks really interesting, but why is it this project's docs? Isn't this something that would be better in the collaborations group docs?

Reply:
Also, what's this for?

Reply:
I suppose I could put it in a collab doc, it's how to set up the notifications that go to the mod-aws channel, which I thought was nicer than emails that will inevitably get lost and is more immediate